app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+  validates :text, presence: true
+
+  after_save :update_comments
+
+  def update_comments_counter
+    post = self.post
```

Comment:
- [ OPTIONAL] I recommend keeping the update comments  counter method private since you used one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_commet_counter` method every time a new comment has been saved to database, so unlike the previous reviewer I think it would be better if you keep as you make It first 💪🏻✅👏🏻💯🔥

Reply:
noted